include/boost/geometry/algorithms/detail/partition.hpp

```diff
@@ -408,17 +421,17 @@ public :
         typename OverlapsPolicy2,
         typename VisitBoxPolicy
     >
-    static inline void apply(Box const& box,
-            IteratorVector1 const& input1,
-            IteratorVector2 const& input2,
-            std::size_t level,
-            std::size_t min_elements,
-            VisitPolicy& visitor,
-            ExpandPolicy1 const& expand_policy1,
-            OverlapsPolicy1 const& overlaps_policy1,
-            ExpandPolicy2 const& expand_policy2,
-            OverlapsPolicy2 const& overlaps_policy2,
-            VisitBoxPolicy& box_policy)
+    static inline bool apply(Box const& box,
+                             IteratorVector1 const& input1,
+                             IteratorVector2 const& input2,
+                             std::size_t level,
+                             std::size_t min_elements,
+                             VisitPolicy& visitor,
+                             ExpandPolicy1 const& expand_policy1,
+                             OverlapsPolicy1 const& overlaps_policy1,
+                             ExpandPolicy2 const& expand_policy2,
+                             OverlapsPolicy2 const& overlaps_policy2,
+                             VisitBoxPolicy& box_policy)
```

Comment:
Yes, it looks better but requires more work unless the environment is able to format automatically somehow. In this case I've done it manually while traversing the file. I agree that I shouldn't have done it as a part of this PR, sorry about that.